go.mod

```
@@ -1,21 +1,104 @@
 module github.com/devfile/registry-operator
 
-go 1.13
+go 1.18
```

```suggestion
go 1.17
```
> `go mod tidy: go.mod file indicates go 1.18, but maximum supported version is 1.17` our integration tests need updating to support golang 1.18, we could cap it at 1.17 until this support is added.

Update it to golang 1.17 instead for now.